api/src/enums/applications/alternate-contact-relationship-enum.ts

```diff
@@ -0,0 +1,7 @@
+export enum AlternateContactRelationship {
```

Comment:
Can we check with product if we should be making this and the HouseholdMemberRelationship enums readable in the CSV export?

Reply:
https://github.com/bloom-housing/bloom/issues/4114

Reply:
And that CSV issue looks existing, but I'll look for a ticket!

Reply:
https://github.com/bloom-housing/bloom/issues/4115